ceph.spec.in

```diff
@@ -62,6 +62,8 @@
 %endif
 %bcond_with seastar
 %bcond_with jaeger
+%bcond_with rbd_rwl
```

Comment:
I know that (historically) the cmake option is just `RBD_RWL`, but might want to be more specific that it's a cache here for clarity.

```suggestion
%bcond_with rbd_rwl_cache
```

Reply:
Updated, and thank you.